app/controllers/service_types_controller.rb

```diff
@@ -4,6 +4,11 @@ def index
   end
 
   def show
-    @business = SearchFacade.a_business(params[:id])
+    @name = params[:name]
+    @image = params[:image]
+    @url = params[:url]
+    @location = params[:location]
+    @phone = params[:phone]
+    @distance = params[:distance].to_f
```

Comment:
This might need some reworking... We have business_id saved to the Favorites table in the BE, so if we use that biz id to route to the show page, we'd have to set up a new route if not using this one.